src/components/systembar/SystemBar.vue

```diff
@@ -0,0 +1,44 @@
+<template>
+  <div class="systemBar">
+    <v-system-bar app color="systemBar" dark>
+      <ProjectsMenu v-if="routeName().includes('project')" />
+      <ModelsMenu v-if="routeName().includes('model')" />
+      <SettingsMenu v-if="routeName() === 'appSettings'" />
+
+      <v-spacer />
+
+      <BackendStatus
+        class="mx-1"
+        :backend="{ id: 'insiteAccess', text: 'Insite' }"
+      />
+      <BackendStatus
+        class="mx-1"
+        :backend="{ id: 'nestSimulator', text: 'NEST' }"
+      />
```

Comment:
We shloud discuss if this code could be more compact, as both are identical except for one line. If we expect to have more than two backends in the future, it would make sense to use a more compact form.